apps/dot_ext/loggers.py

```diff
@@ -106,7 +110,8 @@ def get_session_auth_flow_trace(request):
     if request:
         auth_flow_dict = {}
         for k in SESSION_AUTH_FLOW_TRACE_KEYS:
-            auth_flow_dict[k] = request.session.get(k, None)
+            if k in request.session:
+                auth_flow_dict[k] = request.session.get(k)
```

Comment:
NOTE: Changed this to only get the values if they have a key in the request.session. This will clean up the logging a little bit. However, some of the dashboard panels will need updated to address previous values of "null" vs. the field not being included. Should not be an issue.